File: src/main/java/com/example/appforauslenderamt/entity/PurposeOfStayInGermany.java
package com.example.appforauslenderamt.entity;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.Value;

@Value
@Builder
@Getter
@AllArgsConstructor
public class PurposeOfStayInGermany {

    @JsonProperty("is_changed")
    Boolean isChanged;
    @JsonProperty("explanation")
    String explanation;

}

File: Frontend/src/app/Models/BackendModels/PurposeOfStayInGermany.ts
import { Expose } from "class-transformer"

export class PurposeOfStayInGermany {
    @Expose()
    is_changed: boolean
    @Expose()
    explanation: String

    constructor(isChanged: boolean, explanation: String) {
        this.is_changed = isChanged
        this.explanation = explanation
    }
}
